modyn/trainer_server/internal/trainer/maybe_measure_gpu_ops.py

```diff
@@ -0,0 +1,31 @@
+from types import TracebackType
+from typing import Optional, Type
+
+import torch
+from modyn.common.benchmark import Stopwatch
+
+
+class MaybeMeasureGPUOps:
```

Comment:
nit: similar to my comment below, let's name this `GPUMeasurement` and let it have an enabled flag, to make the name less convoluted

Reply:
done